spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/AbstractConnectionFactoryConfigurer.java

```diff
@@ -40,11 +40,11 @@ protected AbstractConnectionFactoryConfigurer(RabbitProperties properties) {
 		this.rabbitProperties = properties;
 	}
 
-	protected final ConnectionNameStrategy getConnectionNameStrategy() {
+	public final ConnectionNameStrategy getConnectionNameStrategy() {
```

Comment:
I'd prefer to keep this `protected. It should only really be needed by sub-classes.